pom.xml

```diff
@@ -66,6 +67,22 @@
 				<scope>provided</scope>
 			</dependency>
 
+			<dependency>
+				<groupId>org.bahmni.module</groupId>
+				<artifactId>bahmni-migrator</artifactId>
+				<version>${bahmniVersion}</version>
+				<type>jar</type>
+
+			</dependency>
+
+			<dependency>
+				<groupId>org.bahmni.module</groupId>
+				<artifactId>bahmni-extension</artifactId>
+				<version>${bahmniVersion}</version>
+				<type>jar</type>
+
+			</dependency>
+
```

Comment:
openmrs-api and core versions to be upgraded to 2.1.5. change the "openMRSVersion" variable

Reply:
Done